lisp/ess-inf.el

```diff
@@ -512,11 +485,9 @@ This marks the process with a message, at a particular time point."
      (format "\nProcess %s %s at %s\n"
              (process-name proc) message (current-time-string)))))
 
-(defun inferior-ess-make-comint (bufname procname infargs &rest switches)
+(defun inferior-ess-make-comint (bufname procname switches)
   "Make a comint process in buffer BUFNAME with process PROCNAME.
```

Comment:
Which if did you remove? `inferior-ess-make-comint` doesn't have any if removed.

Reply:
This is much easier to see on a local git checkout. Not sure why github is rendering it so poorly.

In `inferior-ess`, there's a line `(setq buf (if switches ...` and I removed the "then" part of the if statement, keeping the "else" part.

Reply:
Ah, I see. Indeed you refactored that infarg argument. Sure, looks right.